bokehjs/src/coffee/models/layouts/layout_dom.coffee

```diff
@@ -56,11 +56,7 @@ class LayoutDOMView extends BokehView
 
   bind_bokeh_events: () ->
     @listenTo(@model, 'change', @render)
-
-    if @model.sizing_mode == 'fixed'
-      @listenToOnce(@model.document.solver(), 'resize', @render)
-    else
-      @listenTo(@model.document.solver(), 'resize', @render)
+    @listenTo(@model.document.solver(), 'resize', () => @render())
```

Comment:
Let's make a small style guide a priority after this release. Not something huge and comprehensive but, at least that collects a few important bits of wisdom like this (e.g. I was not aware of this).